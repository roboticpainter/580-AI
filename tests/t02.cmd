 tests/t02.in 0.1 NS NE W N
