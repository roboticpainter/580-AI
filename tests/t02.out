0.096930986595 4 19 60 
0.549905531590 20 
0.889846542909 28 
0.748565744533 29 
